src/main/java/org/mule/module/xml/internal/util/FileSchemeCorrectionOutputUriResolver.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.xml.internal.util;
+
+import javax.xml.transform.Result;
+
+import net.sf.saxon.lib.StandardOutputResolver;
+import net.sf.saxon.trans.XPathException;
+
+public class FileSchemeCorrectionOutputUriResolver extends StandardOutputResolver {
+
+  private static final String PATH_URI_SCHEME = "file:";
+  private static final String HAS_URI_SCHEME_REGEX = "^[a-zA-Z]+:.+$";
+
+  @Override
+  public Result resolve(String href, String base) throws XPathException {
+    // TODO: Check this!
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-core-extensions-mule-xml-module&pullRequest=29&issues=AWilgRHdotIXnmsUpHVj&open=AWilgRHdotIXnmsUpHVj)

[ISSUE_KEY]: AWilgRHdotIXnmsUpHVj